core/server/worker/pom.xml

```diff
@@ -109,7 +109,10 @@
       <artifactId>hamcrest</artifactId>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-testing</artifactId>
```

Comment:
if this is supposed to be only for testing, add the test scope similar to the dependency above?

Reply:
fixed